shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -338,7 +358,279 @@ public void run() {
   }
 
   @Keep
-  final class ImageTextureRegistryEntry implements TextureRegistry.ImageTextureEntry {
+  final class ImageRawSurfaceTexture
```

Comment:
Do you plan to use this for TLHC too?

Reply:
Yes, A follow on CL will make TLHC mode use this new SurfaceProducer texture.